### ISSUE TITLE
Change in document impacts

### ISSUE THREAD

**Issue by Gangadhar-B** [MicrosoftDocs/architecture-center#2714]
[Enter feedback here]
Hi there, 

Seems the documentation has been changed recently and the header of Azure SQL Database has been renamed to Azure SQL Managed Instance and that caused my development strategy to change completely. I have a screenshot of this documentation stating R/Python is supported for Azure SQL Database from last December. So does it mean all that was mentioned for Azure SQL Database for supporting R/Python is no longer supported now? Any plans in the future?

Regards,
Gangadhar(gaims1@gmail.com)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2808e74b-c325-7b50-093a-b777473864b5
* Version Independent ID: e1f632cb-92ef-958e-5bf4-d250108e1c8d
* Content: [R developer's guide to Azure - R programming - Azure Architecture Center](https://docs.microsoft.com/en-us/azure/architecture/data-guide/technology-choices/r-developers-guide#azure-sql-database)
* Content Source: [docs/data-guide/technology-choices/r-developers-guide.md](https://github.com/microsoftdocs/architecture-center/blob/master/docs/data-guide/technology-choices/r-developers-guide.md)
* Service: **architecture-center**
* Sub-service: **cloud-fundamentals**
* GitHub Login: @AnalyticJeremy
* Microsoft Alias: **jepeach**

**Comment by david-stanford** [comment]
Thanks for reporting - this issue is under review. This is a Microsoft Internal DevOps Bug ID # <a href=https://ceapex.visualstudio.com//pnp/_workitems/edit/389815/>389815</a>

**Comment by AnalyticJeremy** [comment]
@Gangadhar-B, you are correct.  Originally, the Machine Learning Services (which are still in public preview) were slated to be part of the [Azure SQL Database](https://azure.microsoft.com/services/sql-database/) service, and the "R Developer's Guide to Azure" document reflected that.

However, we have since decided to move this feature to be part of Azure SQL Managed Instance, and we have updated the "R Developer's Guide to Azure" document to reflect this change.

You can read more about the Machine Learning Services here: https://docs.microsoft.com/en-us/azure/azure-sql/managed-instance/machine-learning-services-overview
If your solution design requires Machine Learning Services, you will need to change to the SQL MI service.

**Comment by alexhart11** [comment]
@EdPrice-MSFT 
Please close this PR; content in OneNote

**Comment by EdPrice-MSFT** [comment]
As @AnalyticJeremy mentioned, this article has been updated. Thank you for the feedback!
